aptos-node/src/utils.rs

```diff
@@ -70,3 +71,36 @@ pub fn set_aptos_vm_configurations(node_config: &NodeConfig) {
         AptosVM::set_processed_transactions_detailed_counters();
     }
 }
+
+pub fn ensure_max_open_files_limit() {
+    const REQUIRED: u64 = 100_000;
+
+    if !rlimit::Resource::NOFILE.is_supported() {
+        warn!("rlimit setting not supported on this platform. Won't ensure.");
+        return;
+    }
+
+    let (soft, mut hard) = match rlimit::Resource::NOFILE.get() {
+        Ok((soft, hard)) => (soft, hard),
+        Err(err) => {
+            warn!(error = ?err, "Failed getting RLIMIT_NOFILE. Won't ensure.");
```

Comment:
Should you tell the user what kind of limit they need, or in this case it's not supported at all?

Reply:
yeah less likely to hit this branch or anyone reads this warning but doesn't harm to log an extra number.